tasks/php-fpm-centos.yml

```diff
@@ -5,14 +5,14 @@
 - debug: msg="==> Setting phg-fpm on CentOS system"
 
 # Setting  PHP7.
-- name: create socket directory for ceotns 6
+- name: create socket directory for CentOS 6
   file: path=/run/php-fpm/ state=directory mode=0755
   when: ansible_distribution_major_version == "6"
 
 - name: Disable port listen
   lineinfile: >
     dest=/etc/php-fpm.d/www.conf
-    state=present regexp='^listen = 127.0.0.1:9000'
+    state=present regexp='^;?listen = 127.0.0.1:9000'
```

Comment:
OK, I fix it at https://github.com/chusiang/php7.ansible.role/commit/01842798f4411824b3ba9ece90833b1575c20ee8.